app/lib/forms/events/schedule_retry.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Forms
+  module Events
+    class ScheduleRetry
+      include Formify::Form
+      attr_accessor :event
+
+      validates_presence_of :event
+
+      def save
+        with_advisory_lock_transaction(:events, event) do
+          validate_or_fail
+            .and_then { schedule_retry }
+            .and_then { success(event) }
+        end
+      end
+
+      private
+
+      def notify_provider
+        redis = Redis.new
+        key = 'webhooks/provider_last_notified_at'
```

Comment:
Should this be specific to the organization that you're retrying for? Otherwise, how would you know who was the last organization notified?

Reply:
@mattgmarcus this is sent to the provider (e.g. ModernTreasury).  The event ID is also included in that email.  It's not organization-specific, because if the webhook endpoint is down, it's likely down for all organizations.

Reply:
Ah got it